packages/next-dashboard/src/components/Fader.js

```diff
@@ -30,6 +30,9 @@ function Fader<El: string = 'div'>({
       ]
         .filter(c => c)
         .join(' ')}
+      style={{
+        height: '100%',
+      }}
```

Comment:
Wait I forgot this was on the next-dashboard repo and not a project repo... this is probably a bad idea as Fader is meant to be generic and usable in any style context. You probably want to add this as a passthroughable prop instead of applying this everywhere.